### PR TITLE
Ruby Edge Chrome Fixes

### PR DESCRIPTION
**Pull request by titusfortner**
This gets the tests working for the latest version of MicrosoftEdge on both Mac & Windows.

I just merged to trunk other updates to guards related to windows & mac tests and code to get Safari/STP working correctly. I'm doing this as a PR instead of a commit to trunk so we're on the same page with a couple changes here.

1. `Edge` is now aliased to `EdgeChrome` and not `EdgeHtml`; and `Capabilities#edge` returns values for `EdgeChrome`
2. We have unit tests for both `EdgeChrome` & `EdgeHtml` with those classes specifically, but we're only running integration tests for `Edge`
3. Removed build/task for remote-edge-test since 3.x server doesn't work with Edge Chrome (I'll make it all correct with a Selenium 4 server PR)
4. The rest of this is just updating the guards for things that work now with the Chrome implementation.

### CODE REVIEW

File: rb/build.desc [hunk 1]
```diff
@@ -278,16 +279,17 @@ ruby_test(name = "unit",
...
     "spec/unit/selenium/*_spec.rb",
   ],
-  include = ["rb/spec/unit", "build/rb/lib"],
+  include = [
+    "rb/spec/unit",
+    "build/rb/lib"
+  ],
   deps = [
     ":chrome",
-    ":common",
```
Comment (twalpole) [inline rb/build.desc]: Have common, remote, and support gone away? or are they inherited now?

File: rb/spec/integration/selenium/webdriver/window_spec.rb [hunk 2]
```diff
@@ -75,7 +75,7 @@ module WebDriver
         expect(new_pos.y).to eq(target_y)
       end
 
-      it 'gets the rect of the current window', only: {browser: %i[edge edge_chrome firefox ie chrome safari safari_preview]} do
+      it 'gets the rect of the current window', only: {browser: %i[edge firefox ie chrome safari safari_preview]} do
```
Comment (twalpole) [inline rb/spec/integration/selenium/webdriver/window_spec.rb]: What browsers wouldn't this work for?? If there is one maybe `except` makes more sense than `only`

File: rb/spec/integration/selenium/webdriver/window_spec.rb [hunk 3]
```diff
@@ -86,7 +86,7 @@ module WebDriver
         expect(rect.height).to be >= 0
       end
 
-      it 'sets the rect of the current window', only: {browser: %i[edge_chrome firefox ie chrome safari safari_preview]} do
+      it 'sets the rect of the current window', only: {browser: %i[edge firefox ie chrome safari safari_preview]} do
```
Comment (twalpole) [inline rb/spec/integration/selenium/webdriver/window_spec.rb]: Same question as above

**Review by twalpole — APPROVED** [review]
Just added a couple of small comments but overall 👍

File: rb/build.desc (hunk 1 above)
Comment (titusfortner) [reply rb/build.desc]: Each browser inherits common, which inherits everything from support & remote. Seemed unnecessary to repeat it everywhere when nothing was using it.

File: rb/spec/integration/selenium/webdriver/window_spec.rb (hunk 2 above)
Comment (titusfortner) [reply rb/spec/integration/selenium/webdriver/window_spec.rb]: Good catch, I'll fix

**Pull request merged**